src/lustre/lustreAstDependencies.ml

```diff
@@ -394,6 +394,7 @@ let rec get_node_call_from_expr: LA.expr -> (LA.ident * Lib.position) list
   | LA.NArityOp (_, _, es) -> List.flatten (List.map get_node_call_from_expr es)
   | LA.ConvOp (_, _, e) -> get_node_call_from_expr e
   | LA.CompOp (_, _, e1, e2) -> (get_node_call_from_expr e1) @ (get_node_call_from_expr e2)
+  | LA.ChooseOp (_, _, e) -> get_node_call_from_expr e
```

Comment:
This should be handled as a node call.

Reply:
Still pending based on if we commit to moving choose op desugaring earlier in pipeline